src/util/RouterUtil.js

```diff
@@ -146,11 +146,13 @@ function (Okta, OAuth2Util, Util, Enums, BrowserFeatures, Errors, ErrorCodes) {
         successData.session = {
           token: res.sessionToken,
           setCookieAndRedirect: function (redirectUrl) {
-            CourageUtil.redirect(sessionCookieRedirectTpl({
+            var url = sessionCookieRedirectTpl({
               baseUrl: router.settings.get('baseUrl'),
               token: encodeURIComponent(res.sessionToken),
               redirectUrl: encodeURIComponent(redirectUrl)
-            }));
+            });
+
+            RedirectUtil.redirectTo(url);
```

Comment:
We should add a comment here and in the other usage on 140 describing why we're doing a form get here rather than a normal redirect (once we've renamed the methods to be more clear as per @haishengwu-okta's suggestion).